app/models/order_item.rb

```diff
@@ -1,4 +1,14 @@
 class OrderItem < ApplicationRecord
 	belongs_to :order
 	belongs_to :item
+
+	def self.top_three_items
+		item_hash = self.group(:item_id).count
+		popular_items = item_hash.sort_by {|key, value| value}.to_h
```

Comment:
Nice to have this method return the items sorted by popularity, could be useful on another page possibly.

Reply:
How are we determining that an item is popular? What I mean is that it seems it's just grabbing the first three items that are in the orderItems table.